class.jetpack-debugger.php

```diff
@@ -60,6 +81,33 @@ public static function jetpack_debug_display_handler() {
 		$debug_info .= "\r\n" . esc_html( "SITE_URL: " . site_url() );
 		$debug_info .= "\r\n" . esc_html( "HOME_URL: " . home_url() );
 
+		$debug_info .= "\r\n";
+		require_once JETPACK__PLUGIN_DIR . 'sync/class.jetpack-sync-modules.php';
+		$sync_module = Jetpack_Sync_Modules::get_module( 'full-sync' );
+
+		$debug_info .= "\r\n". sprintf( esc_html__( 'Jetpack Sync Full Status: `%1$s`', 'jetpack' ), print_r( $sync_module->get_status(), 1 ) );
```

Comment:
Is there any way we could get human readable times in here as well? This isn't very useful to most people:

```
[started] => 1469788563
[queue_finished] => 1469788563
[sent_started] => 1469788564
[finished] => 1469788564
```